app/helpers/layout_helper.rb

```diff
@@ -98,6 +98,30 @@ def javascript(*args)
     content_for(:javascripts) { javascript_include_tag(*args) }
   end
 
+  def javascript_include_tag(*params, **kwargs)
+    # Workaround for overriding javascript load with webpack_asset_paths, should be removed when webpack_asset_paths is removed
+    if kwargs[:source] == "webpack_asset_paths"
+      kwargs[:webpacked]
+    else
+      super(*params, **kwargs)
+    end
+  end
+
+
+  def webpack_asset_paths(plugin_name, extension: 'js')
+    # @deprecated Previously provided by webpack-rails
```

Comment:
I meant to put it above the method. We don't use [yardoc](https://yardoc.org/) that much, but it's a good standard.